README.rst

```diff
@@ -49,6 +49,88 @@ You can run all of the tox environments using the command ``tox``.
 
 Flocker is also tested using `continuous integration`_.
 
+Cloudformation Deployment
+-------------------------
+
+* Generate a Cloudformation template:
+
+```
+$ python ./admin/installer/cloudformation.py > flocker-cloudformation-template.json
+```
+
+* Validate the template:
+
+```
+$ aws cloudformation validate-template --template-body file://flocker-cloudformation-template.json
```

Comment:
I think this may still need work. (The template is too long now for verification :cry: )
```
flocker-review-3 ♦ ~/dev/flocker-review (flocker-cloudformation-FLOC-3709)$ aws cloudformation validate-template --template-body file://flocker-cloudformation-template.json 2>&1| tail
                        "IpProtocol": "udp",
                        "ToPort": "65535"
                    }
                ]
            },
            "Type": "AWS::EC2::SecurityGroup"
        }
    }
}
' at 'templateBody' failed to satisfy constraint: Member must have length less than or equal to 51200
flocker-review-3 ♦ ~/dev/flocker-review (flocker-cloudformation-FLOC-3709)$
```